src/grt/src/fastroute/src/maze.cpp

```diff
@@ -1247,6 +1247,13 @@ void FastRouteCore::reInitTree(const int netID)
   delete[] sttrees_[netID].edges;
 
   Tree rsmt;
+  const float net_alpha = stt_builder_->getAlpha(nets_[netID]->db_net);
+  // if failing tree was created with pd, fall back to flute with fluteNormal
```

Comment:
This should definitely log a warning so that users are aware that their design is entering a less optimal path. We should also mention some reasons why PD might fail to generate a tree so that the user is empowered to attempt to fix it.

Reply:
The problems are more internal/algorithmic than anything a user can fix.